### PR TITLE
Update release notes for start of Aug

### PR DESCRIPTION
**Pull request by thebenrob**
#### Purpose
Bring release notes up to date for the beginning of August.

#### Changes
Added release notes for 5/21, 5/29, and 7/30

### CODE REVIEW

File: api/release_notes/2019-release-notes.md [hunk 1]
```diff
@@ -31,10 +57,10 @@ rules/:rule_id?include=rule_components
 
 ### 3) Rule Components belong to Properties now
 
-Rule Components used to exist as children of a Rule.  That made it impossible to reuse the same components across multiple rules.  In order to enable us to add this capability in the future, Rule Components need to move out from under a Rule and live at the Property level.
+Rule Components used to exist as a child of a Rule.  That made it impossible to reuse the same components across multiple rules.  In order to enable us to add this capability in the future, Rule Components need to move out from under a Rule and live at the Property level.
```
Comment (perlmonger42) [inline api/release_notes/2019-release-notes.md]: Did you mean to change this back? or maybe you're just accidentally overwriting the change?

**Review by perlmonger42 — CHANGES_REQUESTED** [review]
`Rule Components used to exist as a child of a Rule.` mixes a plural `Rule Components` with singular `child`.

Otherwise, looks good.

File: api/release_notes/2019-release-notes.md [hunk 2]
```diff
@@ -4,6 +4,32 @@ title: 2019 Release Notes
...
+`reactor-releaser` is available on [npm](https://www.npmjs.com/package/@adobe/reactor-releaser).  This tool will help you transition your extension packages from `development` to `private`.  No more Postman scripts if you don't want to use them.
+
+## 2019-05-21
+
+### 1) Attributes is optional when you supply a meta block
+
+If you are passing a `meta` block on a PATCH request, any `attributes` that you pass on the same request are not required and will be ignored.  This change will prevent modifying a resource and transitioning it to a new state at the same time.  This will allow you to get a resource back from the API, and transition its state without having to explicitly remove the `attributes` block before you PATCH it back.  This change affects the following resources.
+
+* Libraries - cannot be modified and transitioned to a new state at the same time
+* Extensions, Rules, and Data Elements - cannot be modified and revised at the same time
```
Comment (spencerroan) [inline api/release_notes/2019-release-notes.md:27]: `hosts` can have `meta: {action: 'test'}` without attributes and cannot be modified and revised at the same time.
Updating the examples is on my list of things to do Once purge retries is done.

File: api/release_notes/2019-release-notes.md (hunk 1 above)
Comment (spencerroan) [reply api/release_notes/2019-release-notes.md]: I think `children` is grammatically correct.

Comment (thebenrob) [reply api/release_notes/2019-release-notes.md]: Seeing some weirdness, I'm sure it's user error.  Changed to children.

File: api/release_notes/2019-release-notes.md (hunk 2 above)
Comment (thebenrob) [reply api/release_notes/2019-release-notes.md:27]: Added the Hosts example to the bullets.

**Comment by thebenrob** [comment]
So, I think I've addressed all the feedback.  In the diffs, I still see the removal of "children", but it's fixed locally, and local says the branch is up to date.  And if I look at the Files tab, it doesn't show the change there, so I think it's done?  Anyway, please re-review?

**Review by spencerroan — APPROVED** [review]

**Review by perlmonger42 — APPROVED** [review]

**Pull request merged**